lib/rules/web/admin_console/4.6/overview.xyaml

```diff
@@ -216,4 +216,63 @@ group_by_label:
     dropdown_menu_item: <label>
   action: click_groupby_dropdown_button
   action: click_dropdown_menu_item
-
+click_workloads_tab:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: click
```

Comment:
No need to add this new rule since user is already on Workloads page

Reply:
@yapei The context focus is on filter name input box and hence the hot keys are not working. Hence I had to click the workloads link one more time to take the context focus away from the filter name box.

Reply:
@schituku  Good, got it